src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -272,6 +273,10 @@ def _execute_command(kwargs):
         try:
             op = get_op_handler(operation)
             result = op(client, **kwargs) if client else op(**kwargs)
+
+            if expose_no_wait and kwargs.get('raw', None):
+                return None #return None for 'no-wait'
```

Comment:
So if 'raw' not in kwargs, we essentially ignore the --no-wait parameter?
Will this be a common thing, if so, consider adding logger.warning message like 'Ignoring no wait because ...'.